### PR TITLE
WT-7865 Disable timeout assert while waiting for eviction to quiesce prior to RTS and test

### PR DESCRIPTION
**Pull request by kommiharibabu**
Enable this assert and the test once the reason behind the active transactions
that are still clearing the cache for more than two minutes in an idle system of 
no more operations.

### CODE REVIEW

File: src/txn/txn_rollback_to_stable.c [hunk 1]
```diff
@@ -1672,7 +1672,13 @@ __rollback_to_stable(WT_SESSION_IMPL *session, bool no_ckpt)
     if (retries == WT_RTS_EVICT_MAX_RETRIES) {
         WT_ERR(__wt_msg(
           session, "timed out waiting for eviction to quiesce, running rollback to stable"));
-        WT_ASSERT(session, false && "Timed out waiting for eviction to quiesce prior to rts");
+        /*
+         * FIXME: WT-7877 RTS fails when there are active transactions running in parallel to it.
+         * The above loop of waiting them to finish is not efficient in some scenarios where the
```
Comment (hbokhari) [inline src/txn/txn_rollback_to_stable.c:1677]: The above loop of waiting them to finish is not efficient -> waiting in a loop for eviction to quiesce is not efficient

**Review by hbokhari — APPROVED** [review]
LGTM

File: src/txn/txn_rollback_to_stable.c (hunk 1 above)
Comment (kommiharibabu) [reply src/txn/txn_rollback_to_stable.c]: Thanks, fixed.

**Review by raviprakashgiri29 — APPROVED** [review]
LGTM

**Pull request merged**